pkg/util/tracing/crdbspan.go

```diff
@@ -375,9 +378,10 @@ func (s *crdbSpan) recordFinishedChildrenLocked(children []tracingpb.RecordedSpa
 
 		s.mu.recording.finishedChildren = append(s.mu.recording.finishedChildren, children...)
 	} else {
-		for _, c := range children {
-			for _, e := range c.StructuredRecords {
-				s.recordInternalLocked(&e, &s.mu.recording.structured)
+		for c := range children {
```

Comment:
[mega nit] `c` looks like it'd be of type tracingpb.RecordedSpan but it's an index.

Reply:
renamed to ci